src/materials/PhysicalPathTracingMaterial.js

```diff
@@ -187,6 +187,11 @@ export class PhysicalPathTracingMaterial extends MaterialBase {
 
 							}
 
+							// alphaMap
+							if ( material.alphaMap != -1 ) {
+								albedo.a *= texture2D( textures, vec3( uv, material.alphaMap ) ).x;
+							}
```

Comment:
nit: can we add extra newlines in the if block and space after the minus in `- 1` to match the others?

Reply:
Surely! Done!

BTW we should add a `.prettierrc` file to the repo because else everyones different Prettier settings will cuase unnecessary large changes

Reply:
There's an eslintrc file but I still need to update a few rules. Either way this shader code is in a string so a linter won't catch it :(